contracts/bridges/scroll/ScrollBridgeReceiver.sol

```diff
@@ -4,10 +4,12 @@ pragma solidity 0.8.15;
 import "../SweepableBridgeReceiver.sol";
 import "./IScrollMessenger.sol";
 
+/// @title Scroll Bridge Receiver
+/// @notice Contract to facilitate message passing from the Comet protocol to Scroll's bridge
```

Comment:
nit: "Contract that processes messages passed from Compound governance using the Scroll bridge"